indigo_app/templates/indigo_api/task_detail.html

```diff
@@ -102,6 +102,35 @@ <h5>{{ task.title }}</h5>
     </div>
 
     <div class="col-2">
+      <h6>Assigned to</h6>
+      <ul class="list-unstyled">
+        {% if task.assigned_to %}
+          <li>{% user_profile task.assigned_to %}</li>
+        {% else %}
+          <li><em>Unassigned</em></li>
+        {% endif %}
+      </ul>
+
+      {% if task.potential_assignees %}
+        <form method="POST">
+          {% csrf_token %}
+          {% if task.assigned_to %}
+            <button class="btn btn-outline-danger mb-3" type="submit" data-confirm="Are you sure you want to unassign this task?" formaction="{% url 'unassign_task' place=place.place_code pk=task.pk %}">Unassign</button>
+          {% endif %}
+          <h6>Assign to</h6>
```

Comment:
I think the "Assign to" heading can be moved to be the dropdown toggle button. So you click "Assign to" and the dropdown is a list of users.

Reply:
I'll update this